### DISCUSSION TITLE
Utilise admin.className for array, row and block fields

### DISCUSSION THREAD

**Discussion by GeorgeyB** [payloadcms/payload#503] [Feature Requests & Ideas]
Could we have array fields use the class name provided to their `admin.className` configurations? That option seems to do nothing currently. This also applies to row and block field types. This would allow for granular styling for individual fields of those types.

In my case, I'm looking to have a "slim" array field that has a slightly different look to other array fields.

Big fan of your work on Payload by the way 👌 love the developer focus that other headless CMS systems seem to be a little wavy on.

**Comment by DanRibbens** [comment]
Thank you very much @GeorgeyB. We'll get your PR out in the next release shortly.
Let us know if you need anything else.

**Reply by jmikrut** [reply]
Update - this has been deployed in `0.15.4`!

**Reply by DanRibbens** [reply]
Just a heads up—we had a bug in yesterdays release that is now fixed in `0.15.5`.
